ui/src/app/pages/login/sign-in.tsx

```diff
@@ -127,127 +127,177 @@ interface SignInState {
   termsOfServiceVersion?: number;
 }
 
-export const SignInReact = fp.flow(withServerConfig(), withWindowSize())(
```

Comment:
GitHub is kinda mangling the diff here. The key change made was to extract the inner class and turn it into a top-level component called SignInReactImpl. This allows us to refer to that class from the unit test, which this PR uses to verify that the set of sign-in steps is correct when invitation key verification is turned off.

Reply:
The diff looks much cleaner if you go "gear menu --> hide whitespace changes", or this link: https://github.com/all-of-us/workbench/pull/3110/files?utf8=%E2%9C%93&diff=unified&w=1

Reply:
it's mangled even with but at least it highlights the differences